src/app/pages/query/query.component.ts

```diff
@@ -609,7 +638,7 @@ export class QueryComponent implements AfterViewInit, OnInit, OnDestroy {
       return dotPath.split('.').reduce((obj, key) => obj[key], obj);
     }
     
-    Object.entries(this.filters).forEach(([key, filter]) => {
+    [...this.filters.entries()].forEach(([key, filter]) => {
```

Comment:
`this.filters` is a Map, not an Object, so the old `Object.entries` wasn't behaving as expected

there are other places in this file where I made similar changes for the same reason